docs/intro.rst

```diff
@@ -28,6 +28,7 @@ pages for each source:
 We also publish SQLite databases containing relatively pristine versions of our more
 difficult to parse inputs, especially the old Visual FoxPro (DBF, pre-2021) and new XBRL
 data (2021+) published by FERC:
+
 * `FERC Form 1 (DBF) <https://s3.us-west-2.amazonaws.com/pudl.catalyst.coop/dev/ferc1.sqlite>`__
 * `FERC Form 1 (XBRL) <https://s3.us-west-2.amazonaws.com/pudl.catalyst.coop/dev/ferc1_xbrl.sqlite>`__
 * `FERC Form 2 (XBRL) <https://s3.us-west-2.amazonaws.com/pudl.catalyst.coop/dev/ferc2_xbrl.sqlite>`__
```

Comment:
I'm in favor of keeping as is because the "Available Data" section in the docs is more concise and links directly to the s3 bucket and data docs pages.

I generally think it's ok to have some duplicative information in the readme and docs because there is a decent chance users who aren't interested in the code skip straight to the documentation which is more data consumer friendly. I'd rather have people read duplicative information in the README and intro page than a data consumer miss information.

Reply:
Fair enough.